app/main.py

```diff
@@ -2,0 +1,22 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[0] == "mv":
+        old_name, new_name = command_list[1:]
+        if "/" not in new_name:
+            pass
+        else:
+            new_name_split = new_name.split("/")
+            path = new_name_split[0]
+            for index in range(1, len(new_name_split)):
+                if os.path.exists(path) is False:
+                    os.mkdir(path)
```

Comment:
The directory creation logic is flawed. The path should be updated before calling `os.mkdir(path)`. Consider using `os.path.join` to build the path correctly .